inc/saiport.h

```diff
@@ -74,6 +74,23 @@ typedef enum _sai_port_oper_status_t
 
 } sai_port_oper_status_t;
 
+/**
+ * @brief Attribute bitmap data for #SAI_PORT_ATTR_ERROR_STATUS
+ */
+typedef enum _sai_port_error_status_t
+{
+    /** No errors */
+    SAI_PORT_ERROR_STATUS_CLEAR = 0,
+
+    SAI_PORT_ERROR_STATUS_MAC_LOCAL_FAULT = 1,
+
+    SAI_PORT_ERROR_STATUS_MAC_REMOTE_FAULT = 2,
+
+    SAI_PORT_ERROR_STATUS_FEC_SYNC_LOSS = 4,
+
+    SAI_PORT_ERROR_STATUS_FEC_LOSS_ALIGNMENT_MARKER = 8
+} sai_port_error_status_t;
+
```

Comment:
There is already an enum sai_port_err_status_t that has similar status'. Can we reuse it?

Reply:
@harshitgulati18 thanks for pointing out. `sai_port_err_status_list_t` is a list of various port error which needs to be traversed to find out which ERROR is set. As the list grows it becomes less efficient whereas in this PR the proposal is to use 64-bit bitmap of errors which is more efficient. I guess 64 bit is sufficient to cover port related errors...